README.md

```diff
@@ -1,2 +1,50 @@
-# react-todo
-This is a simple CRUD web app build with react and sass. The todo list allows user to keep track on his(her) tasks and he(she) can crate new todo, delete todo, mark completed tasks and also edit and update an item. All the tasks will be saved locally using `localStorage`.
+# React TodoList
+
+The TodoList is a simple web application that help users to manage their tasks,
+the way that user can create a new task, can edit and save it again, user can check it as completed
+or delete the tasks.
+
+## see the project
```

Comment:
- [ Optional] Your readme file is very professional. However, I would like to suggest you add a screenshot of your to-do list app to make it more professional.